### ISSUE TITLE
Can we have a demo of Vi-Ui ?

### ISSUE THREAD

**Issue by Mathieu2301** [vitta-health/Vi-Ui#21]
I won't download and use your project if i can't see what it looks like...

**Comment by jeancarlos** [comment]
Hi,

At the moment you can see how each component each looks in the documentation: https://vitta-health.github.io/Vi-Ui/

The documentation is in portuguese, but it shouldn't be a problem if you just want to see how it looks.

We have plans to build a complete exemple right before the 1.0 release (around September) and after the translation for the documentation to english. But I'm focusing mostly at ironing out bugs and improving the existing components right now.

Thank you for your feedback.

**Comment by jeancarlos** [comment]
Leaving the issue open until we have a complete demo.

**Comment by Elvisley** [comment]
Hi,

I'll develop this demo

**Comment by taciomedeiros** [comment]
@jeancarlos The documentation doen't load!

**Comment by gambardellawill** [comment]
Why did you guys merge my branch? That code wasn't ready to be merged?

**Comment by jeancarlos** [comment]
@gambardellawill I merge everything with two approvals, title it as "WIP" (Work in Progress) so I know it's not ready, next time.

**Comment by jeancarlos** [comment]
@taciomedeiros it's a bug. Fixing it.

**Comment by taciomedeiros** [comment]
@Elvisley Do you had some progress with this issue? What do you think that this example should have?

**Comment by jeancarlos** [comment]
@taciomedeiros probably not. LOL. The issue is a necessity, but no one can tackle it right now.

**Comment by jeancarlos** [comment]
Closed by accident. Sorry.

**Comment by Elvisley** [comment]
@taciomedeiros  No, the last time we talked about this topic the idea was to have something similar to the bootstrap demo site. With usage example of each component, and a very simple administrative panel.
